build/config.cmake

```diff
@@ -159,6 +123,46 @@ IF (${ESCARGOT_OUTPUT} STREQUAL "shared_lib")
     SET (ESCARGOT_LDFLAGS_ANDROID ${ESCARGOT_LDFLAGS_ANDROID} -shared)
 ENDIF()
 
+IF (NOT DEFINED ESCARGOT_LIBICU_SUPPORT)
+  SET (ESCARGOT_LIBICU_SUPPORT ON)
+ENDIF()
+
+# LIBICU SUPPORT
+IF (${ESCARGOT_LIBICU_SUPPORT} STREQUAL "ON")
+    SET (ESCARGOT_DEFINITIONS_LINUX ${ESCARGOT_DEFINITIONS_LINUX} -DENABLE_ICU)
+    SET (ESCARGOT_DEFINITIONS_DARWIN ${ESCARGOT_DEFINITIONS_DARWIN} -DENABLE_ICU)
+    SET (ESCARGOT_DEFINITIONS_TIZEN ${ESCARGOT_DEFINITIONS_TIZEN} -DENABLE_ICU)
+
+    # LINUX LIBRARIES
+    IF (${ESCARGOT_ARCH} STREQUAL "x64" OR ${ESCARGOT_ARCH} STREQUAL "x86" OR ${ESCARGOT_ARCH} STREQUAL "arm")
+        PKG_CHECK_MODULES (ICUI18N REQUIRED icu-i18n)
+        PKG_CHECK_MODULES (ICUUC REQUIRED icu-uc)
+        SET (ESCARGOT_LIBRARIES_LINUX ${ESCARGOT_LIBRARIES_LINUX} ${ICUI18N_LIBRARIES} ${ICUUC_LIBRARIES})
+        SET (ESCARGOT_INCDIRS_LINUX ${ESCARGOT_INCDIRS_LINUX} ${ICUI18N_INCLUDE_DIRS} ${ICUUC_INCLUDE_DIRS})
+        SET (ESCARGOT_CXXFLAGS_LINUX ${ESCARGOT_CXXFLAGS_LINUX} ${ICUI18N_CFLAGS_OTHER} ${ICUUC_CFLAGS_OTHER})
+    ENDIF()
+
+    # DARWIN LIBRARIES
+    IF (${ESCARGOT_ARCH} STREQUAL "x64")
+        PKG_CHECK_MODULES (ICUI18N REQUIRED icu-i18n)
```

Comment:
Same as above. It needs to be wrapped with `IF(${ESCARGOT_HOST} STREQUAL "darwin")`.